developers/NewProject.md

```diff
@@ -1,46 +1,30 @@
-
-
-
-
 # New Project Checklist
 
+So, you've been assigned a new project? Great ! \
+Here are the simple steps to get you started. 
+
 ## Architecture
-Your project should be divided in multiple separated components: 
 
-- one or multiple API
-- a web-client
+We work using a ["Micro-Services"](https://en.wikipedia.org/wiki/Microservices) approach. \
+To follow it, your project should be divided in multiple separated components: 
+
+- One or multiple API
+- A web-client
 - A mobile application
+- ...
 
 Start by taking a moment to think how you will divide your project. 
-The point should be to maximum code-reuse across multiple projects for a client.
-
-## Setup Walkthrough (TODO)
-AirTable
-1. Create project in AirTable “Overview” base
-https://airtable.com/tblxH4r1B2L0o6QN4/viwlE7u1JPJYpXQL7
-2. Create a base for your project by duplicating “scrum_template” base
-https://airtable.com/tblSw7z987FQ7tCLU/viw3nOPVnJPOGH0ft
-3. Fill the “components” table, and define your first “sprint”.
-Slack
-1. If this is the first project for this client, create a channel named “r-<CLIENT>” 
-“CLIENT” must be rigorously the same as the one used in AirTable
-2. Create a channel name “p-<PROJECT>”
-“PROJECT” must be rigorously the same as the one used in AirTable
-BitBucket
-Create project in BitBucket
-https://bitbucket.org/account/projects/create?owner=nextmoov
-Create a repository for every component of your project :
-https://bitbucket.org/repo/create?owner=nextmoov
-BitBucket + Slack Integration
-Add a new configuration to the BitBucket integration in Slack
-It should post to r-<CLIENT> channel created earlier.
-Copy the value of the webhook URL.
- https://nextmoov.slack.com/apps/new/A0F7VRDPE-bitbucket
-
-For every repository you created for your project, add this webhook. 
-On BitBucket, go to Repository > Settings > Webhooks > Add
-	Name : Slack
-	URL : The one you copied in step 1
+The point should be to maximize code-reuse across multiple projects.
+
+## Create a repository for any component
+
+- For a backend component (API,...), use our Backend Template : [nextmoov-template-backend](https://github.com/nextmoov/nextmoov-template-backend)
+- For a frontend component, use our Frontend Template : [nextmoov-template-frontend](https://github.com/nextmoov/nextmoov-template-frontend)
```

Comment:
Aren't those repo private ? is that intended ? or do we plan to make those public eventually ?